plugin-modernizer-core/src/main/resources/recipe_data.json

```diff
@@ -0,0 +1,12 @@
+{
+  "recipes": {
+    "PomParser": {
+      "fqcn": "io.jenkins.tools.pluginmodernizer.core.extractor.PomParser",
+      "artifactCoordinates": "io.jenkins.plugin-modernizer:plugin-modernizer-core:999999-SNAPSHOT"
+    },
+    "AddPluginsBom": {
```

Comment:
Improvement also for later is to always use FQDN for recipes, not only the simple class name